package.json

```diff
@@ -63,7 +63,7 @@
     "@pagopa/io-react-native-crypto": "^1.0.1",
     "@pagopa/io-react-native-http-client": "1.0.5",
     "@pagopa/io-react-native-integrity": "^0.3.1",
-    "@pagopa/io-react-native-jwt": "^1.4.0",
+    "@pagopa/io-react-native-jwt": "^2.1.0",
```

Comment:
Remember to commit the `Podfile.lock` too!

Reply:
Right, committed in 9d39f206a7d0bd96843663d18c731397bee3287a.